packages/react/src/components/UIShell/SideNav.tsx

```diff
@@ -14,35 +23,63 @@ import { keys, match } from '../../internal/keyboard';
 // TO-DO: comment back in when footer is added for rails
 // import SideNavFooter from './SideNavFooter';
 
-const SideNav = React.forwardRef(function SideNav(props, ref) {
-  const {
+// const translations = {
+//   'carbon.sidenav.state.open': 'Close',
+//   'carbon.sidenav.state.closed': 'Open',
+// } as const;
+
+// type TranslationId = keyof typeof translations;
+
+// type Translation = typeof translations[TranslationId];
+
```

Comment:
```suggestion
```

Reply:
These depend on this removed code btw:
https://github.com/mrkjdy/carbon/blob/f1eab8039fb175743d8deb9079123ba0f29cab06/packages/react/src/components/UIShell/SideNav.tsx#L34-L35

https://github.com/mrkjdy/carbon/blob/f1eab8039fb175743d8deb9079123ba0f29cab06/packages/react/src/components/UIShell/SideNav.tsx#L54-L55